.github/workflows/continuous-integration.yml

```
@@ -20,7 +20,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [12, 14, 16, 17]
+        node-version: [12, 14, 16, 18]
```

Node.js v17 is still supported, hence we should continue testing it until it EOLs in June 2022. It isn't  a long time from now, but still not EOL yet nonetheless.

```suggestion
        node-version: [12, 14, 16, 17, 18]
```

---

But v17 is not an LTS version and never will be. Should we support him? These versions are discouraged for production.